spec/stream.spec.js

```diff
@@ -1,7 +1,7 @@
 import { Stream } from "../src/stream";
 import { mockApiClient, mockApiClientRequestPromise } from "./mocks/mock-api-client";
 
-describe("Stream",() => {
+describe("Stream", () => {
```

Comment:
I see a bunch of formatting changes in this PR. We should have a project-specific formatting configuration so that this doesn't happen accidentally and so that formatting-only changes stop happening. I can probably handle adding that to `master` before merging this.